libsel4camkes/src/dma.c

```diff
@@ -784,6 +784,9 @@ static void dma_cache_op(void *cookie UNUSED, void *addr UNUSED,
         ZF_LOGE("Could not perform cache op");
         return;
     }
+    if (frame->cached == 0) {
```

Comment:
This might be a bit of nitpicking, but
* Is the cache attribute now ARM specific or generic?
* should the cap always be checked first?
* wouldn't it be better is this prints a "not supported" warning for each certain cache operation if this is not implemented and otherwise just do the operation even if this is uncached. The user might have a reason for this after all that we don't question here as long as it it no forbidden.